internal/v1/handler.go

```diff
@@ -1215,3 +1215,69 @@ func closeBody(body io.Closer) {
 		logrus.Errorf("closing response body failed: %v", err)
 	}
 }
+
+func (h *Handlers) GetOscapProfiles(ctx echo.Context, distribution Distributions) error {
+	switch distribution {
+	case Rhel8:
+		fallthrough
+	case Rhel84:
+		fallthrough
+	case Rhel85:
+		fallthrough
+	case Rhel86:
+		fallthrough
+	case Rhel87:
+		fallthrough
+	case Rhel88:
+		fallthrough
+	case Rhel8Nightly:
+		return ctx.JSON(http.StatusOK, DistributionProfileResponse{
+			XccdfOrgSsgprojectContentProfileAnssiBp28Enhanced,
+			XccdfOrgSsgprojectContentProfileAnssiBp28High,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Intermediary,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Minimal,
+			XccdfOrgSsgprojectContentProfileCis,
+			XccdfOrgSsgprojectContentProfileCisServerL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL2,
+			XccdfOrgSsgprojectContentProfileCui,
+			XccdfOrgSsgprojectContentProfileE8,
+			XccdfOrgSsgprojectContentProfileHipaa,
+			XccdfOrgSsgprojectContentProfileIsmO,
+			XccdfOrgSsgprojectContentProfileOspp,
+			XccdfOrgSsgprojectContentProfilePciDss,
+			XccdfOrgSsgprojectContentProfileStandard,
+			XccdfOrgSsgprojectContentProfileStig,
+			XccdfOrgSsgprojectContentProfileStigGui,
+		})
+	case Rhel9:
+		fallthrough
+	case Rhel90:
+		fallthrough
+	case Rhel91:
+		fallthrough
+	case Rhel92:
+		fallthrough
+	case Rhel9Nightly:
+		return ctx.JSON(http.StatusOK, DistributionProfileResponse{
+			XccdfOrgSsgprojectContentProfileAnssiBp28Enhanced,
+			XccdfOrgSsgprojectContentProfileAnssiBp28High,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Intermediary,
+			XccdfOrgSsgprojectContentProfileAnssiBp28Minimal,
+			XccdfOrgSsgprojectContentProfileCis,
+			XccdfOrgSsgprojectContentProfileCisServerL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL1,
+			XccdfOrgSsgprojectContentProfileCisWorkstationL2,
+			XccdfOrgSsgprojectContentProfileCui,
+			XccdfOrgSsgprojectContentProfileE8,
+			XccdfOrgSsgprojectContentProfileHipaa,
+			XccdfOrgSsgprojectContentProfileIsmO,
+			XccdfOrgSsgprojectContentProfileOspp,
+			XccdfOrgSsgprojectContentProfilePciDss,
+			XccdfOrgSsgprojectContentProfileStig,
+			XccdfOrgSsgprojectContentProfileStigGui,
+		})
+	default:
+		return echo.NewHTTPError(http.StatusBadRequest, "No profile for the specified distribution")
```

Comment:
It would be nice to have `centos-8` and `centos-9` too

Reply:
They should be considered as `rhel8` and `rhel9` right ?

Reply:
There is weirdly a difference between `centos-8` and `rhel-8`. Same for `centos-9` and `rhel-9`

Reply:
All done :)